docker/release/entrypoint-tools.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+
+echo "Starting java ${JAVA_NET_OPTS} ${JAVA_MEM_OPTS} ${JAVA_GC_OPTS} ${JAVA_EXTRA_OPTS} -cp /alephium-tools.jar $@"
+exec java ${JAVA_NET_OPTS} ${JAVA_MEM_OPTS} ${JAVA_GC_OPTS} ${JAVA_EXTRA_OPTS} -cp /alephium.jar $@
```

Comment:
Should it be `alephium-tools.jar`?

Reply:
Yes if we want this image to be specific about prune storage. We can also add more commands to this image in the future. 

Right now for pruning we can run something like this:
```
docker run -it -v /mnt/disks/alephium-data:/alephium-home/.alephium  alephium/alephium-tools:2.6.1 org.alephium.tools.PruneStorage
```